docs/PLUGINS.md

```diff
@@ -198,9 +198,11 @@ Simplest option for you to test JCasC compatibility in your plugin is to introdu
 
 Add the Configuration as Code plugin as a test dependency in your pom.xml:
 
+[LATEST_RELEASE](https://github.com/jenkinsci/configuration-as-code-plugin/releases/latest)
```

Comment:
might be a badge url you can use for this?

Reply:
Will this do? :sweat_smile: 
![latest release](https://img.shields.io/github/release/jenkinsci/configuration-as-code-plugin.svg)

Reply:
What about:
```
[![latest release](https://img.shields.io/github/release/jenkinsci/configuration-as-code-plugin.svg)](https://github.com/jenkinsci/configuration-as-code-plugin/releases/latest)
```

Reply:
Same badge but linking to the release

Reply:
I'll guess I have to badge the readme too :laughing: